src/uslscore/uslscore.h

```diff
@@ -87,7 +87,6 @@
 #include <uslscore/USTypedPtr.h>
 #include <uslscore/USTypeID.h>
 #include <uslscore/USUnion.h>
-#include <uslscore/USUnique.h>
```

Comment:
Never used?

Reply:
Yes, I tried to look for references everywhere but didn't find any. Also, was the only class that was giving issues with libc++ so just decided to remove it completely

Reply:
sg